### PR TITLE
Don't use ginkgo.focus for ci-reboot

### PR DESCRIPTION
**Pull request by j3ffml**
focus is whitelist, we want to run unflaky, unslow tests along with
reboot to verify cluster functionality before/after.

### CODE REVIEW

**Comment by jlowdermilk** [comment]
@k8s-bot e2e test this

**Comment by ixdy** [comment]
Given that the gce-reboot project runs only the reboot tests, maybe we want a comment explaining why this is different?

**Comment by jlowdermilk** [comment]
Actually, we really should be doing the same for gce as well. Testing only the reboot tests in isolation does a poor job of verifying that the reboot does not disrupt cluster functionality.

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -688,8 +688,8 @@ case ${JOB_NAME} in
           ${GKE_DEFAULT_SKIP_TESTS[@]:+${GKE_DEFAULT_SKIP_TESTS[@]}} \
           ${REBOOT_SKIP_TESTS[@]:+${REBOOT_SKIP_TESTS[@]}} \
           ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
-          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
-          ) --ginkgo.focus=Reboot"}
+          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}}
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: should we keep the \ at the end of this line? (not sure if it matters, but that's the pattern other builds follow)

Comment (jlowdermilk) [inline hack/jenkins/e2e.sh]: Ah thank, that omission actually broke the script I think.

**Comment by jlowdermilk** [comment]
@k8s-bot test this

**Comment by jlowdermilk** [comment]
@k8s-bot test this

**Pull request merged**